### ISSUE TITLE
Idea: Add option to create a project with an application.yml

### ISSUE THREAD

**Issue by bdemers** [spring-io/start.spring.io#486]
I would love to be able to create a new project that contains an `applicaiton.yml` instead of an `applciation.properties` file.

Maybe as an option? `properties=yaml`

Or maybe `whiteSpaceBugs=true` :trollface: 
Joking aside, we commonly give instructions to rename app.props to app.yml after creating a new project and then provide example configuration in YAML format.

I can understand the desire to keep the number of permutations down, so feel free to close this issue!

Thanks again!

**Comment by snicoll** [comment]
> I can understand the desire to keep the number of permutations down,

Yep, that's what happening here. This was raised a few times and I understand the interest but I can't see that happening at the moment. I've flagged this for team attention to see what the rest of the team things. I've also moved that to the library as I don't think this would be 100% on the service-side only.
